actions/class.ItemPreview.php

```diff
@@ -35,7 +35,13 @@ public function forwardMe(){
         $item = new core_kernel_classes_Resource(tao_helpers_Uri::decode($this->getRequestParameter('uri')));
         $lang = DEFAULT_LANG;
         $previewUrl = taoItems_models_classes_ItemsService::singleton()->getPreviewUrl($item, $lang);
+
+        if (null === $previewUrl) {
+            throw new \common_exception_NotFound('Item not found');
```

Comment:
Could you wrap your message in `__('')` our i18n helper. Also it would be better to use exception which extends `common_exception_UserReadableException`